\documentclass[a4paper, 11pt]{article}
\usepackage{comment} % enables the use of multi-line comments (\ifx \fi) 
\usepackage{fullpage} % changes the margin
\usepackage[margin=35pt]{geometry}
\usepackage{graphicx}
\usepackage{microtype} 
\usepackage{titlesec}
\usepackage{color, colortbl}
\usepackage[table]{xcolor} 
\usepackage[numbers,super]{natbib}
\usepackage{bibentry}
\nobibliography*

%\usepackage[most]{tcolorbox}

%\usepackage{fontspec}
%\setmainfont{Georgia} 
\titleformat*{\section}{\large\bfseries}

\begin{document}

\noindent
\large\textbf{Committee Meeting Report} \hfill \textbf{Arjun Biddanda} \\
\normalsize  \hfill Professor: John Novembre  \\
HG Matriculated 2015 \hfill Date: 3/8/18 \\
\noindent\makebox[\linewidth]{\rule{\paperwidth}{0.4pt}}

\large\textbf{\\Progress since last Committee Meeting - October X 2018}
\subsection*{Awards}
% Horizontal line after name; adjust line thickness by changing the '1pt'
\begin{itemize}
    \item ASHG Reviewer's Choice Abstract Award  \hfill 2019 
    \item Genetics and Regulation Training Grant \hfill 2015-2018
\end{itemize} 

\subsection*{Manuscripts in progress}
\begin{itemize}
    \item Drafting: 
		\item Drafting:
\end{itemize}

\subsection*{Oral Presentations}
\begin{itemize}
	\item Midwest Population Genetics Meeting \hfill August 2019 \\ \emph{Linkage Disequilibrium and Haplotype Patterns in Ancient DNA}
	\item Novembre-He-Stephens (NHS) Meeting \hfill June 2019 \\ \emph{Linkage Disequilibrium and Haplotype Patterns in Ancient DNA}   
\end{itemize}

\subsection*{Posters}
\begin{itemize}
	 \item Biddanda A, Steinr\"{u}cken M, Novembre J. Linkage Disequilibrium and Haplotype Patterns in Ancient DNA : Theory + Applications. Poster presented at American Society of Human Genetics 2019
	 \item Biddanda A, Novembre J. Inference and visualization of the geographic distribution for variant sets. Poster presented at American Society of Human Genetics 2018 
\end{itemize}


\subsection*{Teaching Assistantship Requirements Completed}
\begin{itemize}
	\item HGEN 48600 \emph{Fundamentals of Computational Biology: Models and Inference} \hfill Winter 2018\\
	\item HGEN 46900 \emph{Human Variation and Disease} \hfill Spring 2017\\
\end{itemize}


\subsection*{Extracurricular}
\begin{itemize}
	\item  Polsky Center: Spark Fellow \hfill Summer 2019 \\
		Performed due-diligence and market analysis for projects applying for \$10-\$50k through Polsky for early stage funding to expand business opportunities  
	\item  Tutoring Chicago:  \hfill Fall 2019 -  \\
		Tutoring 6th grade students in STEM once a week	
\end{itemize}

% This is where most of the development + planning goes...
\newpage

%\noindent\makebox[\linewidth]{\rule{\paperwidth}{0.4pt}}
\section*{Since my last committee meeting:}

	\textbf{AIM 1} To identify and characterize parent of origin effects on quantitative traits in the Hutterites. \\
	
 \textbf{\\AIM 2a} To identify and characterize parent of origin effects on gene expression in 306 Hutterites.\\
 
 \textbf{\\AIM 2b} To identify and characterize parent of origin and allele specific effects on gene expression in 306 Hutterites.\\ 
 	
%\end{tcolorbox}
\bibliography{report2019}
\bibliographystyle{apalike}

%\begin{thebibliography}{9}
%\bibitem{Gamazon} Gamazon, E. R., Wheeler, H. E., Shah, K. P., Mozaffari, S. V., Aquino-Michaels, K., Carroll, R. J., et al. (2015). A gene-based association method for mapping traits using reference transcriptome data. Nature Genetics, 47(9), 1091–1098. http://doi.org/10.1038/ng.3367
%\bibitem{Cusanovich} Cusanovich, D. A., Caliskan, M., Billstrand, C., Michelini, K., Chavarria, C., De Leon, S., et al. (2016). Integrated analyses of gene expression and genetic association studies in a founder population. Human Molecular Genetics, ddw061. http://doi.org/10.1093/hmg/ddw061
%\bibitem{Dobin} Dobin, A., \& Gingeras, T. R. (2015). Mapping RNA-seq Reads with STAR. Current Protocols in Bioinformatics. 51, 11.14.1–19. http://doi.org/10.1002/0471250953.bi1114s51
%\bibitem{Jun} G. Jun, M. Flickinger, K. N. Hetrick, Kurt, J. M. Romm, K. F. Doheny, G. Abecasis, M. Boehnke,and H. M. Kang, (2012) Detecting and Estimating Contamination of Human DNA Samples in Sequencing and Array-Based Genotype Data, AJHG doi:10.1016/j.ajhg.2012.09.004 (volume 91 issue 5 pp.839 - 848)
%\bibitem{van de Geijn}van de Geijn B, McVicker G, Gilad Y, Pritchard JK. (2015) WASP: allele-specific software for robust molecular quantitative trait locus discovery. Nat Meth. 12:1061-1063. doi:10.1038/nmeth.3582.
%\end{thebibliography}

\end{document}
